samples/qa-a11y/src/views/Panels.js

```diff
@@ -33,6 +33,7 @@ const PanelsView = () => {
 					dataSize={itemList.length}
 					itemRenderer={customItem}
 					itemSize={ri.scale(156)}
+					spotlightId="virtual-list"
```

Comment:
```suggestion
					dataSize={itemList.length}
					id={'virtualList_$' + index}
					itemRenderer={customItem}
					itemSize={ri.scale(156)}
					spotlightId={'virtualList_$' + index}
```

id also need to restore scroll position.

and.. I prefer keep the previous format. (it's not affect the functions. but just a preference)